src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/BaseType.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tizen.Applications.CoreBackend;
+using static Interop.CBApplication;
+using static Tizen.Applications.CoreBackend.DefaultCoreBackend;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    internal class BaseType
```

Comment:
Naming is not good, it is too ambiguous.

Reply:
I changed it into BaseFactory.